scripts/integration_test_contracts.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash -e
+
+# sleep
+# This sleep is the sum of the time when docker is pulled, started,
+# and the height of the node increases to 1 or more.
+sleep 30
+
+# send transaction to node
+cur_path=`pwd`
+chain_dir=${cur_path}"/finschia/template/.finschia"
+artifacts_path=${cur_path}"/../artifacts/"
+
+echo "hogehoge"
+echo ${artifacts_path}
+echo ${chain_dir}
+
+#*** store collection contract ***
+# store contract
+fnsad tx wasm store ${artifacts_path}/collection.wasm --node http://localhost:26658 --home=${chain_dir} --from link146asaycmtydq45kxc8evntqfgepagygelel00h --chain-id simd-testing --keyring-backend test --gas 10000000  -b block -o json -y > tmp.json
```

Comment:
Need prepare const for 26658, link146asaycmtydq45kxc8evntqfgepagygelel00h

Reply:
I prepared const valuables in this commit. Please check it.

[fix: define the valuables with values used in this test](https://github.com/Finschia/finschia-wasm/pull/3/commits/cc1b47fe9a97c070cb48f2c3bb87050565d68b5d)